### PR TITLE
Updates contact information to logos

### PR DESCRIPTION
**Pull request by mmurray22**
Adds logos to the website instead of just links.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -52,11 +52,25 @@ <h1>Michaela Murray's Portfolio</h1>
...
           </form>
-        </div> 
-        <div class="mdc-touch-target-wrapper" style="display:block;">
-      	  <button class="mdc-button mdc-button--outlined" onclick="contact()" style="color:white;">  
-	     <span class="mdc-button__ripple">Contact</span> 
-	  </button>
+        </div>
+	<div id="icons" style="display:block">
+            <ul style="list-style-type:none;margin-left:-42px;"> 
+            <li style="display:inline-block;">
```
Comment (kaveh096) [inline portfolio/src/main/webapp/index.html]: You're adding styles directly to your elements but in other cases you're using css. i'm still not sure what is the right way. Consider checking appropriate guides.

**Review by kaveh096 — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: Hi Kaveh, I checked the style guide, and according to this point here: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns, the style should be completely confined to the css file! I'll address that in my next commit-thanks!

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -11,31 +11,33 @@ body {
...
-   text-align: center;
+form {
+  display:inline-block;
 }
 
-#intro-buttons {
-  display: inline-block;
-  padding: 1%;
+#icon {
+  display:inline-block;
```
Comment (ccondit) [inline portfolio/src/main/webapp/style.css]: [style nit](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops) - and with all of the property names in this file:
> Always use a single space between property and value

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -11,56 +11,70 @@
...
 	    </button>
           </form>
-        </div> 
-        <div class="mdc-touch-target-wrapper" style="display:block;">
-      	  <button class="mdc-button mdc-button--outlined" onclick="contact()" style="color:white;">  
-	     <span class="mdc-button__ripple">Contact</span> 
-	  </button>
+        </div>
+	<div id="icons">
+            <ul >
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html]: some of these elements have a space after the element name and the closing `>`.
`<ul >` should be `<ul>`, etc

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -11,56 +11,70 @@
...
           </form>
-        </div> 
-        <div class="mdc-touch-target-wrapper" style="display:block;">
-      	  <button class="mdc-button mdc-button--outlined" onclick="contact()" style="color:white;">  
-	     <span class="mdc-button__ripple">Contact</span> 
-	  </button>
+        </div>
+	<div id="icons">
+            <ul > 
+            <li >
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html]: Some of this indentation looks off again (the li is at the same indentation level as the ul). But let's not block on that for now...

**Review by ccondit — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: thanks! updated!

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/style.css]: thanks! updated!

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -18,49 +18,63 @@
...
-        <div class="mdc-touch-target-wrapper" style="display:block;">
-      	  <button class="mdc-button mdc-button--outlined" onclick="contact()" style="color:white;">  
-	     <span class="mdc-button__ripple">Contact</span> 
-	  </button>
+        </div>
+	<div id="icons">
+            <ul> 
+            <li> 
+                <a href="https://github.com/mmurray22" target="_blank">
+                    <img id="icon" src="/images/logo-github.svg">
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html]: So obviously the browser can parse this, but it's considered an error to use the same ID for multiple elements. If you're using the same ID to get uniform styling for these elements I would change this to use a CSS class, instead. 
image

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -11,31 +11,33 @@ body {
...
    border: 1px solid black;
+   display: inline-block;
    padding: 10px;
 }
 
-#inner-content {
-   border: 1px solid black;
-   text-align: center;
+form {
+  display: inline-block;
```
Comment (ccondit) [inline portfolio/src/main/webapp/style.css]: Is there a `form` introduced in this PR? If not, let's isolate this change to its related PR.

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (ccondit) [inline portfolio/src/main/webapp/index.html]: Likewise for img elements, per [the style guide](https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback), alternative content should be provided. This is important for making sure that products are accessible to all users.

**Review by ccondit — CHANGES_REQUESTED** [review]
Some of the indentation still looks slightly off, but let's not block on that.

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: ah ok thanks! updated!

File: portfolio/src/main/webapp/style.css (hunk 6 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/style.css]: ah ok-noted! updated!

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: noted! done!

Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: thanks for pointing that out! :)

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -49,25 +49,25 @@ <h1>Michaela Murray's Portfolio</h1>
         <div class="mdc-touch-target-wrapper">
           <form action="/pdf/Michaela_Murray_Resume.pdf" target="_blank">
             <button class="mdc-button mdc-button--outlined">  
-	       <span class="mdc-button__ripple">Resume</span> 
-	    </button>
+	            <span class="mdc-button__ripple">Resume</span> 
+	        </button>
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html]: looks like some indentation changed here - mind reverting?

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -49,25 +49,25 @@ <h1>Michaela Murray's Portfolio</h1>
...
           </form>
         </div>
 	<div id="icons">
             <ul> 
             <li> 
-                <a href="https://github.com/mmurray22" target="_blank">
-                    <img id="icon" src="/images/logo-github.svg">
+                <a href="https://github.com/mmurray22" target="_blank" alt="Github icon">
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html]: The `alt` attribute should apply to the `img` element. The first two links had the alt applied to the anchor, instead.

File: portfolio/src/main/webapp/index.html [hunk 9]
```diff
@@ -49,25 +49,25 @@ <h1>Michaela Murray's Portfolio</h1>
...
-                <a href="https://www.linkedin.com/in/murray22/" target="_blank">
-                    <img id="icon" src="/images/logo-linkedin.svg">
+                <a href="https://www.linkedin.com/in/murray22/" target="_blank" alt="LinkedIn Icon">
+                    <img class="icon" src="/images/logo-linkedin.svg">
                 </a>
             </li> 
             <li> 
                 <a href="mailto:murray22@stanford.edu">
-                    <img id="icon" src="/images/email-24px.svg">
+                    <img class="icon" src="/images/email-24px.svg" target="_blank" alt="Email Icon">
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html]: I don't think `target` is a valid attribute for `img`. Did you mean for this to go on the anchor?

**Review by ccondit — CHANGES_REQUESTED** [review]

**Review by kaveh096 — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 8 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: thanks! updated!

File: portfolio/src/main/webapp/index.html (hunk 9 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html]: yes! updated!

**Review by ccondit — APPROVED** [review]

**Pull request merged**